test/e2e/skywalking-components-with-satellite.yaml

```diff
@@ -55,9 +55,9 @@ metadata:
   name: skywalking-system
   namespace: skywalking-system
 spec:
-  version: 8.9.1
+  version: 9.0.0
```

Comment:
9.0.0 has a critical bug for its internal queue, please use 9.1.0 for testing.

Reply:
Get it.